docker/2diirunner-with-packages/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM maurolepore/2diirunner:1.3.0
+MAINTAINER "Mauro Lepore" maurolepore@gmail.com
+
+RUN apt-get update && apt-get install -y pandoc
+
+# This repository has binaries not just for Windows and Mac, but also for Linux
+RUN echo "options(repos = c(CRAN = 'https://packagemanager.rstudio.com/all/__linux__/bionic/latest'))" >> /root/.Rprofile \
+    # update-dockerfile-packages-start
+    && Rscript -e "install.packages('conflicted')" \
+    && Rscript -e "install.packages('config')" \
+    && Rscript -e "install.packages('countrycode')" \
+    && Rscript -e "install.packages('cowplot')" \
+    && Rscript -e "install.packages('devtools')" \
+    && Rscript -e "install.packages('data.table')" \
+    && Rscript -e "install.packages('dplyr')" \
+    && Rscript -e "install.packages('extrafont')" \
+    && Rscript -e "install.packages('fs')" \
+    && Rscript -e "install.packages('fst')" \
+    && Rscript -e "install.packages('ggforce')" \
+    && Rscript -e "install.packages('ggmap')" \
+    && Rscript -e "install.packages('ggplot2')" \
+    && Rscript -e "install.packages('ggrepel')" \
+    && Rscript -e "install.packages('ggthemes')" \
+    && Rscript -e "install.packages('glue')" \
+    && Rscript -e "install.packages('grid')" \
+    && Rscript -e "install.packages('gridExtra')" \
+    && Rscript -e "install.packages('here')" \
```

Comment:
Not sure, but it might be more efficient to install these packages all in one command. I assume that every time you use `Rscript` a new R console is spun up, which probably has some non-negligible start-up and clean-up overhead

Reply:
Agree with that. My suggestion would be to write a script (`install_packages.R`) that has you dependencies, and call that.

Reply:
@AlexAxthelm, what's a good way to pass such a file install_packages.R to the container? For now I fixed this in an inelegant way like so:

```r
  ...
  && Rscript -e "install.packages(c("pkg1", "pkg2", ...))"